app/models/parcel.rb

```diff
@@ -170,7 +170,8 @@ class Parcel < Ekylibre::Record::Base
 
   validate do
     if given_at
-      errors.add(:given_at, :not_opened_financial_year) unless opened_financial_year?
+      errors.add(:given_at, :not_opened_financial_year) unless opened_financial_year? || financial_year_in_closure_preparation?
```

Comment:
Can you wrap this in a `if(closure_preparation?)/else` instead of `unless ... closure_preparation?` + `if closure_preparation?` ?